src/classes/sequelize-service.ts

```diff
@@ -67,15 +67,15 @@ export class SequelizeService<W, R extends W, C extends {} = {}> extends Service
 
         await this.executeHook(Hook.DID_CREATE, session, this._afterCreate.bind(this));
 
-        return session.getAt(0) as R & C;
+        return session.getAt(0) as R & Pick<C, Compute>;
       });
     });
   }
 
-  public async createMany(objects: W[], options: TCreateOptions<R, C> = {}): Promise<Collection<R & C>> {
+  public async createMany<Compute extends keyof C = undefined>(objects: W[], options: TCreateOptions<R, C, Compute> = {}): Promise<Collection<R & Pick<C, Compute>>> {
```

Comment:
Feels a bit weird, doesn't it?

Reply:
Yeah, it looks super weird, but it does exactly what we want, lol